pkg/asset/installconfig/azure/client.go

```diff
@@ -286,7 +286,9 @@ func (c *Client) GetVMCapabilities(ctx context.Context, instanceType, region str
 	if typeMeta == nil {
 		return nil, fmt.Errorf("not found in region %s", region)
 	}
-
+	if typeMeta.Family != nil && to.String(typeMeta.Family) == "standardDCADSv5Family" {
+		return nil, fmt.Errorf("standardDCADSv5Family is not currently stupported but will be in a future release")
+	}
```

Comment:
I don't think this check should live here, because this function is just for grabbing the VM capabilities. We don't do any validation in this function, so we should move this validation to be with the rest. If this is the only place we get the `typeMeta`, which I suspect is the case, we can just do some minor refactoring to expose `typeMeta` to the calling validation code.

Reply:
I've stuffed the family type into the capabilities.